packages/driver/cypress/e2e/commands/querying/querying.cy.js

```diff
@@ -82,6 +82,12 @@ describe('src/cy/commands/querying', () => {
       cy.get('doesNotExist')
     })
 
+    it('respects null withinSubject', () => {
+      cy.get('#list').within(() => {
+        cy.get('#upper', { withinSubject: null })
```

Comment:
I know this is fixing a v11 -> v12 regression ,but is this a valid use-case of get in a within callback? Seems a little odd 😅 I had always assumed this was intended to narrow down, not remove scope

Reply:
I can imagine a situation like this:

```
Cypress.Commands.add('loginAs', (name) => {
  cy.get('#login-form', { withinSubject: null })
    .find('.username')
    .type(name)
  // ...do rest of login...
})

cy.get('#user-profile').within(() => {
  cy.get('#username').should('eq', '')
  cy.loginAs('fred')
  cy.get('#username').should('eq', 'fred')
})
```

where the custom command wouldn't want to respect any 'parent' `.within()` blocks. Doesn't seem super common, but also not obviously incorrect.